pkgs/development/tools/frink/default.nix

```diff
@@ -0,0 +1,35 @@
+{ pkgs }:
+with pkgs;
+let
+  jar = fetchurl {
```

Comment:
Why?

Just put this as `src`.

Reply:
I wanted to also get the units file and functions file for rlwrap to provide tab completion but I haven't gotten around to it, yet. I'm not sure I ever will and I don't otherwise have strong opinions on it, so sure, if you prefer.